selfdrive/version.py

```diff
@@ -51,35 +51,42 @@ def get_git_remote(default=None):
 origin = get_git_remote()
 branch = get_git_full_branchname()
 
-try:
-  # This is needed otherwise touched files might show up as modified
+if (origin is not None) and (branch is not None):
   try:
-    subprocess.check_call(["git", "update-index", "--refresh"])
-  except subprocess.CalledProcessError:
-    pass
-
-  if (origin is not None) and (branch is not None):
     comma_remote = origin.startswith('git@github.com:commaai') or origin.startswith('https://github.com/commaai')
     tested_branch = get_git_branch() in ['devel', 'release2-staging', 'dashcam-staging', 'release2', 'dashcam']
 
-    dirty = not comma_remote
+    dirty = False
```

Comment:
```suggestion
    dirty = not prebuilt
```

Reply:
I don't think that will matter. If it's not prebuilt the dirty variable will overwritten a few lines below. If the git command fails it will be marked dirty anyway.